projects/storefrontlib/src/cms-components/content/paragraph/paragraph.component.html

```diff
@@ -1 +1 @@
-<p *ngIf="component.data$ | async as data" [innerHTML]="data.content"></p>
+<div *ngIf="component.data$ | async as data" [innerHTML]="data.content"></div>
```

Comment:
It's a breaking change in the DOM (and for any customer styling) so should be documented in our 4_0.md migration docs.

See https://sap.github.io/spartacus-docs/breaking-changes/#avoiding-breaking-changes

Reply:
Good to know! I will add it to the migration docs.